frontend/src/components/Summary/ImplementationRow.tsx

```diff
@@ -49,13 +49,18 @@ const ImplementationRow = ({
       </td>
 
       <td>
-        <button
-          type="button"
-          className="btn btn-sm btn-primary"
-          onClick={() => setShowDetails(true)}
-        >
-          Details
-        </button>
+        {implementation.failedTests +
```

Comment:
Great. Can you wrap this in a `.hasUnsuccessful` property on implementations, this same addition is used in a few places I think.

Reply:
But in the unsuccessful column we just add the `failed tests + errored tests` but here we are adding skippedTests as well.